pkg/cmd/kube-router.go

```diff
@@ -115,7 +115,7 @@ func (kr *KubeRouter) Run() error {
 	nsInformer := informerFactory.Core().V1().Namespaces().Informer()
 	npInformer := informerFactory.Networking().V1().NetworkPolicies().Informer()
 
-	go informerFactory.Start(stopCh)
```

Comment:
removed `go `, as `Start()` method actually launches go routing for each informer an exits